src/main/kotlin/blackjack/domain/card/PlayerCardDeck.kt

```diff
@@ -0,0 +1,35 @@
+package blackjack.domain.card
+
+import blackjack.domain.player.PlayerName
+import java.util.LinkedList
+
+class PlayerCardDeck(
+    private val playerName: PlayerName,
+) {
+
+    private val _cards = LinkedList<Card>()
+    val cards: List<Card> = _cards
+
+    fun insert(card: Card) {
+        _cards.add(card)
+    }
+
+    fun insertAll(newCards: List<Card>) {
+        _cards.addAll(newCards)
+    }
+
+    fun isNotEmpty(): Boolean {
+        return isEmpty().not()
+    }
```

Comment:
```suggestion
    fun isNotEmpty(): Boolean {
        return _cards.isNotEmpty()
    }
```
아실수도 있으시겠지만, isNotEmpty라는 함수도 있습니다. 😄

Reply:
좋은 의견 감사합니다 :) 

말씀 주신 것 처럼 isNotEmpty()라는 좋은 함수가 존재하고 있는데요~! PlayerCardDeck 내에서의 isEmpty() <-> isNotEmpty()가 서로 반대 관계임을 명확히 표현하고자 한 의도였습니다. 

반영을 할까 하다가, 반영하기 전 의견을 한번 공유해봐도 좋을 것 같아 의견 먼저 공유 드립니다 :)

Reply:
오호 그런 의도가 있으셨군요! 👍 

> isNotEmpty()가 서로 반대 관계임을 명확히 표현하고자 한 의도였습니다.

이 부분이 마음에 드네요. 💯 
다만, 저는 개인적으로는 가급적이면 언어단에서 제공해주는 함수를 활용하는 편입니다. 😅